llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgTransform/LinalgTransformOps.td

```diff
@@ -105,13 +105,35 @@ def TileOp : Linalg_Transform_Operation<"tile",
 
   let arguments = (ins PDL_Operation:$target,
                    DefaultValuedAttr<I64ArrayAttr, "{}">:$sizes,
-                   DefaultValuedAttr<I64ArrayAttr, "{}">:$interchange,
-                   DefaultValuedAttr<BoolAttr, "false">:$scalarize_dyn_dims);
+                   DefaultValuedAttr<I64ArrayAttr, "{}">:$interchange);
   let results = (outs PDL_Operation:$tiled_linalg_op,
                       Variadic<PDL_Operation>:$loops);
 
   let hasCustomAssemblyFormat = 1;
-  let hasVerifier = 1;
+
+  let extraClassDeclaration = [{
+    ::mlir::LogicalResult apply(
+        ::mlir::linalg::transform::TransformResults &transformResults,
+        ::mlir::linalg::transform::TransformState &state);
+  }];
+}
+
+def ScalarizeOp : Linalg_Transform_Operation<"scalarize",
+    [TransformOpInterface]> {
```

Comment:
I think one could add the TargetableSingleOperandTransformOpTrait here and implement the applyToOne method only.

Reply:
Indeed, thanks!